CONTRIBUTING.md

```diff
@@ -1,79 +1,81 @@
-
-Quick steps and ideas of how you can become a contributor!
-
-1.  Code Code Code (and file Pull Request)
-2.  Create design assets or style guide revsions
-3.  Submit updates and improvements to the documentation.
-4.  Submit articles and guides which are also part of the documentation.
-5.  Join the Slack to help and interact with grommet users
-6.  Help a Grommet designer or developer by answering questions on
-    StackOverflow, Slack and GitHub.
-7.  Report bugs and proposing new features by filing issues on Github or ask in our Slack community
-8.  Share with us exciting projects using grommet on Slack
-
-## Contributing
+Thank you for reaching the contribution page and showing the true gremlin in
+you! In Grommet we do believe that the more the merrier – Welcome! Thank you for
+making this step of joining and contributing to our community, and for helping
+us make Grommet the best tool for streamlining the way you develop apps. You
+came to the right place to start your contribution! Follow the guidelines and
+let us know if we can help with anything else.
+
+## Grommet Projects
+
+Grommet is divided into several projects, the following are notable:
+
+- [grommet] – the primary Grommet 2.X project is actively developed and
+  contributions are more than welcome! Be sure to check the [good first issues].
+- [grommet-icons] – iconography for Grommet and React.js.
+- [react-desc] – add a schema to your React components based on React
+  [`PropTypes`][prop-types].
+- [design-kit] – the Grommet Design Kit provides a set of sticker sheets and
+  templates to help bootstrap your design process.
+
+## You can Become a Contributor
+
+Afterall, that’s why you’re here, right?
+Quick steps and ideas of how you can contribute to Grommet:
+
+1. Code, code, code… (and make a Pull Request).
+1. Create design assets or style guide revisions.
+1. Submit updates and improvements to the documentation.
+1. Submit articles and guides which are also part of the documentation.
+1. Join the [Slack community] to interact with and help Grommet users.
+1. Help a Grommet designer or developer by answering questions on
+   [Stack Overflow], [Slack][slack community], or [GitHub][grommet issues].
+1. Report bugs and propose new features by [filing issues on
+   GitHub][grommet issues], or come talk to us and fellow contributors in our
+   [Slack community] about your issue or idea.
+1. Share with us exciting projects using Grommet in our [Slack community].
+
+## How to Contribute
 
 The best way to collaborate with the project contributors is through the Grommet
 organization on GitHub: <https://github.com/grommet>.
 
-You are invited to contribute new features, fixes, or updates, large or small; we
-are always thrilled to receive pull requests, and do our best to process them as
-fast as we can.
+You are invited to contribute new features, fixes, or updates – large or small.
+We are always thrilled to receive pull requests, and do our best to process them
```

Comment:
On that note, thank you ;)

Reply:
Yeah … is this a meta-contribution to contribute to the Contribution guidelines… ? 😆

Reply:
lol